wolfssl/wolfcrypt/ed448.h

```diff
@@ -86,6 +86,8 @@ struct ed448_key {
 #ifdef WOLFSSL_ASYNC_CRYPT
     WC_ASYNC_DEV asyncDev;
 #endif
+    void *heap;
+    wc_Shake sha;
```

Comment:
I don't see any header trap for `WOLFSSL_SHAKE256` in the header? Guessing we require it with ed448, however someone who isn't using configure might accidentally mess this up. Either add `WOLFSSL_SHAKE256` at top with warning or wrap this.